package.json

```diff
@@ -40,23 +40,35 @@
     "cross-fetch": "^3.1.4"
   },
   "devDependencies": {
+    "@azure/storage-blob": "^12.11.0",
     "@itwin/build-tools": "^3.0.0",
-    "@itwin/certa": "^3.0.0",
     "@itwin/core-bentley": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/oidc-signin-tool": "^3.2.0",
+    "@itwin/eslint-plugin": "^3.2.6",
+    "@itwin/oidc-signin-tool": "^3.4.1",
     "@types/chai": "^4.1.4",
-    "@types/mocha": "^8.2.2",
-    "@types/node": "^14.14.31",
+    "@types/chai-as-promised": "^7.1.5",
+    "@types/mocha": "^8.2.3",
+    "@types/node": "^14.18.22",
+    "@types/sinon": "^10.0.13",
+    "axios": "^0.27.2",
     "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "chromium": "^3.0.3",
+    "copyfiles": "^2.4.1",
     "dotenv": "^10.0.0",
     "dotenv-expand": "^5.1.0",
-    "eslint": "^7.11.0",
+    "eslint": "^7.32.0",
+    "inversify": "^6.0.1",
+    "isomorphic-fetch": "^3.0.0",
     "mocha": "^8.3.2",
+    "nyc": "^15.1.0",
+    "puppeteer": "^15.5.0",
+    "reflect-metadata": "^0.1.13",
+    "sinon": "^14.0.0",
     "source-map-loader": "^1.0.0",
```

Comment:
`source-map-loader` might not be required. Try removing it and check if everything still works. This package is a webpack plugin and this package does not use webpack anymore.

Reply:
Removed and everything is working fine.

Reply:
Then please do remove it. `npm uninstall source-map-loader`.